awsiot/iotshadow.py

```diff
@@ -935,20 +977,25 @@ def __init__(self, *args, **kwargs):
     def from_payload(cls, payload):
         # type: (typing.Dict[str, typing.Any]) -> ErrorResponse
         new = cls()
-        val = payload.get('clientToken')
-        if val is not None:
-            new.client_token = val
-        val = payload.get('code')
-        if val is not None:
-            new.code = val
-        val = payload.get('message')
-        if val is not None:
-            new.message = val
-        val = payload.get('timestamp')
-        if val is not None:
-            new.timestamp = datetime.datetime.fromtimestamp(val)
+        if 'clientToken' in payload:
+            new.clientToken = payload['clientToken']
+            new.clientToken_is_nullable = new.clientToken is None
```

Comment:
we don't want to do it this new way for EVERY member, only members tagged "nullIsValid" in the model